app/models/brand.rb

```diff
@@ -182,8 +184,47 @@ def in_trial?
     !!subscription.try(:trial?)
   end
 
+  def delete_account
```

Comment:
Also [here](https://github.com/signalplus-dev/signalplus/blob/master/db/schema.rb#L205) we'd want to make sure that these don't collide. Let's make that index compound on `deleted_at` too.